src/Controls/GlobeControls.js

```diff
@@ -206,25 +216,25 @@ class GlobeControls extends THREE.EventDispatcher {
         this.maxDistance = options.maxDistance || ellipsoidSizes.x * 8.0;
 
         // Limits to how far you can zoom in and out ( OrthographicCamera only )
-        this.minZoom = 0;
-        this.maxZoom = Infinity;
+        this.minZoom = options.minZoom || 0;
+        this.maxZoom = options.maxZoom || Infinity;
 
         // Set to true to disable this control
         this.rotateSpeed = options.rotateSpeed || 0.25;
 
         // Set to true to disable this control
-        this.keyPanSpeed = 7.0; // pixels moved per arrow key push
+        this.keyPanSpeed = options.keyPanSpeed || 7.0; // pixels moved per arrow key push
 
         // How far you can orbit vertically, upper and lower limits.
         // Range is 0 to Math.PI radians.
         // TODO Warning minPolarAngle = 0.01 -> it isn't possible to be perpendicular on Globe
-        this.minPolarAngle = THREE.MathUtils.degToRad(0.5); // radians
-        this.maxPolarAngle = THREE.MathUtils.degToRad(86); // radians
+        this.minPolarAngle = options.minPolarAngle ? THREE.MathUtils.degToRad(options.minPolarAngle) : THREE.MathUtils.degToRad(0.5);
+        this.maxPolarAngle = options.maxPolarAngle ? THREE.MathUtils.degToRad(options.maxPolarAngle) : THREE.MathUtils.degToRad(86);
```

Comment:
Maybe you could simplify with 
```js
this.minPolarAngle = THREE.MathUtils.degToRad(options.minPolarAngle ?? 0.5);
```
And same with `maxPolarAngle` ?

Reply:
Ok, done and rebased :)